jarviscli/plugins/teachme.py

```diff
@@ -0,0 +1,420 @@
+from nltk.tokenize import word_tokenize as wt, sent_tokenize as st
+from nltk.corpus import wordnet
+from nltk.stem import WordNetLemmatizer
+from collections import Counter
+import nltk
+import numpy as np
+import matplotlib.pyplot as plt

```

Comment:
Why this import?

Reply:
We intended to have some advanced features including some graphs but we decided that it wouldn't be very useful for a CLI but we forgot to remove some import, they should now be a bit cleaner.